README.md

```diff
@@ -1,2 +1,401 @@
+dependencies.
+
+### Ubuntu
+
+1. Get the Zephyr source code:
+
+      ```bash
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        --mf zephyr/west.yml \
+        ~/zxa
+      cd ~/zxa
+      west update
+      source zephyr/zephyr-env.sh
+      ```
+
+1. Export a `Zephyr CMake package`. This allows CMake to
+   automatically load boilerplate code required for building Zephyr
+   applications.
+
+      ```bash
+      west zephyr-export
+      ```
+
+1. Install additional Python dependencies specified by any module
+
+      ```bash
+      pip3 install --user -r ~/zxa/zephyr/scripts/requirements.txt
+      ```
+
+1. Configure `west` to place build output in 
+   project-board-specific folders
+
+      ```bash
+      west config \
+        build.dir-fmt ${ZEPHYR_BASE}/../build/{source_dir}/{board}
+      ```
+
+### macOS
+
+1. Get the Zephyr source code:
+
+      ```bash
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        --mf zephyr/west.yml \
+        ~/zxa
+      cd ~/zxa
+      west update
+      source zephyr/zephyr-env.sh
+      ```
+
+1. Export a `Zephyr CMake package`. This allows CMake to
+   automatically load boilerplate code required for building Zephyr
+   applications.
+
+      ```bash
+      west zephyr-export
+      ```
+
+1. Install additional Python dependencies specified by any module
+
+      ```bash
+      pip3 install -r ~/zxa/zephyr/scripts/requirements.txt
+      ```
+
+1. Configure `west` to place build output in 
+   project-board-specific folders
+
+      ```bash
+      west config \
+        build.dir-fmt ${ZEPHYR_BASE}/../build/{source_dir}/{board}
+      ```
+
+### Windows
+
+1. Get the Zephyr source code:
+
+      ```bat
+      cd %HOMEPATH%
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        --mf zephyr\west.yml \
+        zxa
+      cd zxa
+      west update
+      source zephyr\zephyr-env.sh
+      ```
+
+1. Export a `Zephyr CMake package`. This allows CMake to
+   automatically load boilerplate code required for building Zephyr
+   applications.
+
+      ```bat
+      west zephyr-export
+      ```
+
+1. Install additional Python dependencies specified by any module
+
+      ```bat
+      pip3 install -r HOMEPATH%\zxa\zephyr\scripts\requirements.txt
+      ```
+
+1. Configure `west` to place build output in 
+   project-board-specific folders
+
+      ```bat
+      west config build.dir-fmt ${ZEPHYR_BASE}\..\build\{source_dir}\{board}
+      ```
+
+
+## Build and Run the zxa_example_project
+
+Build the `zxa_example_project` with
+`west build`, changing ``<your-board-name>``
+appropriately for your board:
+
+### Ubuntu
+
+1. Build projects directly from the command line
+
+      ```bash
+      cd ~/zxa
+      west build -p auto -b native_posix \
+        example-application/zephyr/projects/zxa_example_project
```

Comment:
This should be more than enough:

`west build -p auto -b reel_board app`

Let's have a single `app` folder inside the repo with an application. We can build on that later.

Do not use `native_posix` because that only works on Linux

Reply:
@carlescufi Having already worked for months with a more elaborate example, I am convinced we will be **far** better off separating the concept of a 'project/test/sample' from the application-level service code.  It makes sense to me that an `app/` folder contains application-level business logic that should be able to be pulled in to multiple projects.

Also, "later" will happen very soon.

Reply:
I honestly disagree. `example-application/zephyr/projects/zxa_example_project` is way too long a path for users that are just getting started here, this is way too much for them to digest in one go. Let us start small.

Reply:
Also, please use the reel_board and not native posix. This is for consistency with what we do in the Zephyr GSG

Reply:
At the beginning of the README it states that this is repository is supplementary to the Zephyr repository, and provides links back to Zephyr documentation.  It is unreasonable to expect people to successfully use this that haven't first gone through GSG and some samples.  With that as a background `example-application/zephyr/projects/zxa_example_project` is no deeper than `samples/subsys/logging/logger` and only two characters longer than `samples/portability/cmsis_rtos_v1/timer_synchronization`.  What makes this way too long?